cvat-ui/src/components/annotation-page/annotation-page.tsx

```diff
@@ -80,17 +80,17 @@ export default function AnnotationPageComponent(props: Props): JSX.Element {
                 <AnnotationTopBarContainer />
             </Layout.Header>
             { workspace === Workspace.STANDARD && (
-                <Layout.Content>
+                <Layout.Content style={{ height: '100%' }}>
```

Comment:
Why not add style to scss file, for example for class selector `ant-layout-content`?

Reply:
This change is suitable not for any ``Layout.Content``.
We can specify additional class, but I believe it isn't necessary right now